### ISSUE TITLE
Excluded files should be excluded in migrated project

### ISSUE THREAD

**Issue by poizan42** [hvanbakel/CsprojToVs2017#190]
cs files not included in source project should have a `<Compile Remove="file.cs" />` entry generated for them in the transformed project.

**Comment by hvanbakel** [comment]
These are the ones that you currently get a warning for, correct? The warning says that they are now included because of the wildcard include (not saying that I disagree).

**Comment by poizan42** [comment]
@hvanbakel Nope, it doesn't says anything. This is the output from running dotnet-migrate-2017 migrate on https://gist.github.com/poizan42/3788073d08eb5b665c353cbe732ec016

```
[08:34:00 INF] No nuspec found, skipping package configuration.
[08:34:00 INF] Packages.config file not found.
[08:34:48 WRN] Could not read from assemblyinfo, no assemblyinfo file found
[08:34:48 INF] Removed 2 include items thanks to Microsoft.NET.Sdk defaults
[08:34:48 INF] Backing up to D:\projects\ExcludedFileTest\Backup
[08:35:10 INF] No nuspec found, skipping package configuration.
[08:35:10 INF] Packages.config file not found.
[08:35:10 WRN] Could not read from assemblyinfo, no assemblyinfo file found
[08:35:10 INF] W020:
ExcludedFileTest.csproj:15: A better way to reference 'Microsoft.CSharp' assembly is using 'Microsoft.CSharp' NuGet package. It will simplify porting to other runtimes.
[08:35:10 INF] W021:
ExcludedFileTest.csproj:14: A better way to reference 'System.Data.DataSetExtensions' assembly is using respective 'System.Data.DataSetExtensions' NuGet package. It will simplify porting to other runtimes and enable possible .NET SDK tooling improvements.
[08:35:10 INF] W021:
ExcludedFileTest.csproj:16: A better way to reference 'System.Net.Http' assembly is using respective 'System.Net.Http' NuGet package. It will simplify porting to other runtimes and enable possible .NET SDK tooling improvements.
[08:35:10 INF] W030:
ExcludedFileTest.csproj:8: Consider migrating to 'portable' debug type, cross-platform alternative to legacy Windows PDBs.
[08:35:10 INF] W030:
ExcludedFileTest.csproj:11: Consider migrating to 'portable' debug type, cross-platform alternative to legacy Windows PDBs.
```

The migrated project does not explicitly exclude the file Excluded.cs so it is included.

**Comment by hvanbakel** [comment]
Merged in #191
